pytest

```diff
@@ -0,0 +1,6 @@
+reset
+pytest -xv ./ -n 4
+pytest --cache-clear --pep8 -m pep8
+pytest --cache-clear --flakes -m flakes
+pytest --cache-clear --mccabe -m mccabe
+radon cc osbrain/ -s -nb
```

Comment:
This file could/should be removed. The user can now easily select what to run without that script.

If you want to run only some tests (having `keyword` in the test name):

    pytest -k keyword

If you want to run just the behavior tests:

    pytest -n 8

If you want to run all tests but for one Python version only (faster):

    tox -e py35

If you want to run all tests for all available Python versions:

    tox